ci/update-packages/update_nginx_blob.sh

```diff
@@ -34,8 +34,27 @@ else
     echo "Importing PGP key ${pgp_key}"
     gpg --import "${pgp_key}"
   done
-  echo "Downloading nginx archive..."
-  wget -O "nginx-${new_nginx_version}.tar.gz"     "https://nginx.org/download/nginx-${new_nginx_version}.tar.gz"
+
+  # archive may not yet be ready immediately for download on nginx.org -> exponential retry
+  attempt=1
+  max_attempts=4
+  retry_delay_sec=300
+  while (( attempt <= max_attempts )); do
+    echo "Downloading nginx archive (attempt $attempt/$max_attempts)..."
+    if ! wget -O "nginx-${new_nginx_version}.tar.gz" "https://nginx.org/download/nginx-${new_nginx_version}.tar.gz"; then
+      echo "Command failed, retrying in $retry_delay_sec seconds"
+      sleep $retry_delay_sec
+      retry_delay_sec=$((retry_delay_sec * 2)) # double the retry delay for the next attempt
+      ((attempt++))
```

Comment:
Should this go below the `if ...`? Currently only 3 attempts are done, after the third failure `attempt` is increased and the `if ...` condition is true.

Reply:
Fixed.